x-pack/plugins/reporting/export_types/printable_pdf/server/lib/layouts/layout.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { kbn_server } from '../../../../../../../../src/server/index';
```

Comment:
`kbn_server` => `KbnServer`

I think this import path will actually break when we create a release because x-pack code gets moved around to a different folder.

@azasypkin - do you know the best way to use a KbnServer type?  I saw there was a `KbnServer` type in the new platform, but not sure if or how we should be referencing it in legacy code.

Reply:
> I think this import path will actually break when we create a release because x-pack code gets moved around to a different folder.

Yeah, this is something we shouldn't use.

> @azasypkin - do you know the best way to use a KbnServer type? I saw there was a KbnServer type in the new platform, but not sure if or how we should be referencing it in legacy code.

@Stacey-Gammon that's a good question, I haven't thought about this yet, _but_ how about using this:
```ts
....
private readonly server: import('../../../../../../../../src/server').KbnServer;
...
```

This won't import module, only type and will be removed by TS during compilation/transpilation. See more details here: https://blogs.msdn.microsoft.com/typescript/2018/05/31/announcing-typescript-2-9/#import-types